File: src/main/java/dev/fynf/randomfood2/entities/ConnectorRepository.java
package dev.fynf.randomfood2.entities;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface ConnectorRepository extends CrudRepository<Connector, Long> {
  List<Connector> findAllByIdNotIn(List<Long> ids);
}

File: src/main/kotlin/dev/fynf/randomfood2/entities/ConnectorRepository.kt
package dev.fynf.randomfood2.entities

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface ConnectorRepository : CrudRepository<Connector?, Long?> {
    fun findAllByIdNotIn(ids: List<Long?>?): List<Connector?>?
}